pkgs/development/roswell/default.nix

```
@@ -0,0 +1,36 @@
+{ lib, stdenv, fetchFromGitHub, curl, autoreconfHook, makeWrapper, sbcl}:
+
+stdenv.mkDerivation rec {
+  pname = "roswell";
+  version = "21.01.14.108";
+  src = fetchFromGitHub {
+    owner = "roswell";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1hj9q3ig7naky3pb3jkl9yjc9xkg0k7js3glxicv0aqffx9hkp3p";
+  };
+
+  configurePhase = ''
+    runHook preConfigure
+    sh bootstrap && ./configure --prefix $out
+    runHook postConfigure
+  '';
+
+  postInstall = ''
+    wrapProgram $out/bin/ros \
+      --add-flags 'lisp=sbcl-bin/system sbcl-bin.version=system' \
+      --prefix PATH : ${lib.makeBinPath [ sbcl ]} --argv0 ros
+  '';
+
+  nativeBuildInputs = [ autoreconfHook makeWrapper ];
+  buildInputs = [ sbcl curl ];
+
+  meta = with lib; {
+    description = "Roswell is a Lisp implementation installer/manager, launcher, and much more!";
```

[CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/e7e72e070b9a82730e97134bf33d6a7e608a6bf3/.github/CONTRIBUTING.md):

> `meta.description` should:
>   - […]
>   - Not have a period at the end.

It's an exclamation mark in your case but I'd extend that guideline to punctuation marks in general.

```suggestion
    description = "Roswell is a Lisp implementation installer/manager, launcher, and much more";
```